JavaOpdracht6/familyTree/src/test/java/PersonTest.java

```diff
@@ -55,23 +55,30 @@ void should_Return_Siblings(){
 
         // Assert
         assertEquals(siblingList, irene.getSiblings());
+
```

Comment:
Je test addSibling niet, want in regel 54 overschrijf je siblings met siblingList.